crates/extensions/tedge_health_ext/src/actor.rs

```diff
@@ -0,0 +1,119 @@
+use async_trait::async_trait;
+use tedge_actors::Actor;
+use tedge_actors::CombinedReceiver;
+use tedge_actors::DynSender;
+use tedge_actors::MessageBox;
+use tedge_actors::ReceiveMessages;
+use tedge_actors::RuntimeError;
+use tedge_actors::RuntimeRequest;
+use tedge_actors::WrappedInput;
+use tedge_api::health::health_status_up_message;
+use tedge_mqtt_ext::MqttMessage;
+use tedge_mqtt_ext::TopicFilter;
+
+type HealthInputMessage = MqttMessage;
+type HealthOutputMessage = MqttMessage;
+
+pub struct TedgeHealthMonitorActor {
```

Comment:
```suggestion
pub struct HealthMonitorActor {
```
I'd drop that `Tedge` prefix just to keep it short, as every actor that we develop is a `Tedge` actor in the end ;)

Reply:
Fixed, but even the directories are prefixed with `tedge`, which may be not necessary.

Reply:
I guess the those directory prefixes are to explicitly label/differentiate between `tedge` and `c8y` extensions.